app/views/expenses/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_with model: expense, url: group_expenses_path(@group) do |form| %>
+  <header class="bg-primary text-white p-2">
+    <nav class="d-flex justify-content-between align-items-center text-center">
+   <%= link_to group_expenses_path do %>
+   <i class="fa-solid fa-chevron-left"></i>
+    <span class="sr-only"></span>
+    <% end %>
+
+    <h2 class="m-0 p-0">New expense</h2>
+    <%= form.submit "Save", class: "nav-link fs-2" %>
+    </nav>
+  </header>
+  <% if expense.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(expense.errors.count, "error") %> prohibited this expense from being saved:</h2>
+
+        <ul>
+          <% expense.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+   <% end %>
```

Comment:
- [ ] Great job on creating the form so far but I think it would be great if you add an input to let the user enter at ;east one category as required so please add it here you can make it in any way you like but you should include it in the form 💯🔥🤩

![image](https://user-images.githubusercontent.com/58553711/235318291-6a365e7a-3295-46b3-bd1a-369d1057d585.png)